### PR TITLE
Address #15

### PR DESCRIPTION
**Pull request by seanturner**
Addresses #15.

### CODE REVIEW

File: draft-ietf-lamps-dilithium-certificates.xml [hunk 1]
```diff
@@ -120,13 +120,16 @@
...
+  AlgorithmIdentifier{ALGORITHM-TYPE, ALGORITHM-TYPE:AlgorithmSet} ::=
+    SEQUENCE {
+      algorithm   ALGORITHM-TYPE.&id({AlgorithmSet}),
+      parameters  ALGORITHM-TYPE.
+                    &Params({AlgorithmSet}{@algorithm}) OPTIONAL
+    }
        </sourcecode>
        <aside>
-       <t>NOTE: The above syntax is from <xref target="RFC5280"/> and matches the version used therein, i.e., the 1988 ASN.1 syntax. See <xref target="RFC5912"/> for ASN.1 copmatible with the 2015 ASN.1 syntax.</t>
+       <t>NOTE: The above syntax is from <xref target="RFC5912"/> and is compatible with the 2021 ASN.1 syntax {{X680}}. See
```
Comment (seanturner) [inline draft-ietf-lamps-dilithium-certificates.xml:131]:
```suggestion
       <t>NOTE: The above syntax is from <xref target="RFC5912"/> and is compatible with the 2021 ASN.1 syntax <xref target="X680"/>}. See
```

File: draft-ietf-lamps-dilithium-certificates.xml [hunk 2]
```diff
@@ -120,13 +120,16 @@
...
-   AlgorithmIdentifier  ::=  SEQUENCE  {
-       algorithm   OBJECT IDENTIFIER,
-       parameters  ANY DEFINED BY algorithm OPTIONAL
-   }
+  AlgorithmIdentifier{ALGORITHM-TYPE, ALGORITHM-TYPE:AlgorithmSet} ::=
+    SEQUENCE {
+      algorithm   ALGORITHM-TYPE.&id({AlgorithmSet}),
+      parameters  ALGORITHM-TYPE.
+                    &Params({AlgorithmSet}{@algorithm}) OPTIONAL
+    }
```
Comment (seanturner) [inline draft-ietf-lamps-dilithium-certificates.xml:128]:
```suggestion
   AlgorithmIdentifier{ALGORITHM-TYPE, ALGORITHM-TYPE:AlgorithmSet} ::=
     SEQUENCE {
       algorithm   ALGORITHM-TYPE.&id({AlgorithmSet}),
       parameters  ALGORITHM-TYPE.
                     &Params({AlgorithmSet}{@algorithm}) OPTIONAL
     }
```

File: draft-ietf-lamps-dilithium-certificates.xml [hunk 3]
```diff
@@ -120,13 +120,16 @@
...
-       algorithm   OBJECT IDENTIFIER,
-       parameters  ANY DEFINED BY algorithm OPTIONAL
-   }
+   AlgorithmIdentifier{ALGORITHM-TYPE, ALGORITHM-TYPE:AlgorithmSet} ::=
+     SEQUENCE {
+       algorithm   ALGORITHM-TYPE.&id({AlgorithmSet}),
+       parameters  ALGORITHM-TYPE.
+                     &Params({AlgorithmSet}{@algorithm}) OPTIONAL
+     }
        </sourcecode>
```
Comment (seanturner) [inline draft-ietf-lamps-dilithium-certificates.xml:129]:
```suggestion
       </artwork>
```

File: draft-ietf-lamps-dilithium-certificates.xml [hunk 4]
```diff
@@ -120,13 +120,16 @@
       <t>This specification uses placeholders for object identifiers until the identifiers for the new algorithms are assigned by NIST.</t>
       <t>The AlgorithmIdentifier type, which is included herein for convenience, is defined as follows: </t>
        <sourcecode type="asn.1">
```
Comment (seanturner) [inline draft-ietf-lamps-dilithium-certificates.xml:122]:
```suggestion
       <artwork type="asn.1">
```

File: draft-ietf-lamps-dilithium-certificates.xml [hunk 5]
```diff
@@ -119,14 +119,17 @@
...
-   }
-       </sourcecode>
+       <artwork type="asn.1">
+   AlgorithmIdentifier{ALGORITHM-TYPE, ALGORITHM-TYPE:AlgorithmSet} ::=
+     SEQUENCE {
+       algorithm   ALGORITHM-TYPE.&id({AlgorithmSet}),
+       parameters  ALGORITHM-TYPE.
+                     &Params({AlgorithmSet}{@algorithm}) OPTIONAL
+     }
+       </artwork>
```
Comment (seanturner) [inline draft-ietf-lamps-dilithium-certificates.xml:129]:
```suggestion
       <sourcecode type="asn.1">
   AlgorithmIdentifier{ALGORITHM-TYPE, ALGORITHM-TYPE:AlgorithmSet} ::=
     SEQUENCE {
       algorithm   ALGORITHM-TYPE."&amp;"id({AlgorithmSet}),
       parameters  ALGORITHM-TYPE.
                     "&amp;"Params({AlgorithmSet}{@algorithm}) OPTIONAL
     }
       </sourcecode>
```

**Comment by seanturner** [comment]
Need to land #14 to get X680 ref.

**Comment by jakemas** [comment]
Fixing conflict. Again @seanturner let me know if this looks good, and I will merge!

**Pull request merged**